### DISCUSSION TITLE
Handycam Preset

### DISCUSSION THREAD

**Discussion by notdigidani** [ntsc-rs/ntsc-rs#132] [Presets]
### @notdigidani Handycam VHS Preset

vhs preset: newest version aviable: **v.2.4** ▶ [dani's vhs preset v2.4.json](https://github.com/user-attachments/files/29681200/dani.s.vhs.preset.v2.4.json)

⬇ **preview** ⬇
> <img width="641" height="481" alt="image" src="https://github.com/user-attachments/assets/40b16027-9326-44c4-866c-db852fd9fd30" />

  
  
  
⬇ **render settings** ⬇
> <img width="409" height="141" alt="Captura de pantalla 2026-02-07 163008" src="https://github.com/user-attachments/assets/4da65381-9430-48f6-906d-63a6446f44ed" />

**all of the previous versions:**

> - **v.1.0**  ▶ [dani's vhs preset.json](https://github.com/user-attachments/files/16193979/dani.s.vhs.preset.json)
> 
> 
> - **v.1.1**  ▶ [dani's vhs preset v1.1.json](https://github.com/user-attachments/files/16283956/dani.s.vhs.preset.v1.1.json)
> 
> 
> - **v.1.2**  ▶ [dani's vhs preset v1.2.json](https://github.com/user-attachments/files/16741689/dani.s.vhs.preset.v1.2.json)
> 
> 
> - **v.1.3**  ▶ [dani's vhs preset v1.3.json](https://github.com/user-attachments/files/16874765/dani.s.vhs.preset.v1.3.json)
> 
> 
> - **v.1.4**  ▶ [dani's vhs preset v1.4.json](https://github.com/user-attachments/files/17002772/dani.s.vhs.preset.v1.4.json)
> 
> 
> - **v.1.5**  ▶ [dani's vhs preset v1.5.json](https://github.com/user-attachments/files/17117471/dani.s.vhs.preset.v1.5.json)
> 
> 
> - **v.1.6**  ▶ [dani's vhs preset v1.6.json](https://github.com/user-attachments/files/17117472/dani.s.vhs.preset.v1.6.json)
> 
> 
> - **v.1.7**  ▶ [dani's vhs preset v1.7.json](https://github.com/user-attachments/files/17117475/dani.s.vhs.preset.v1.7.json)
> 
> 
> - **v.1.8**  ▶ [dani's vhs preset v1.8.json](https://github.com/user-attachments/files/17246501/dani.s.vhs.preset.v1.8.json)
> 
> 
> - **v.1.9**  ▶ [dani's vhs preset v.1.9.zip](https://github.com/user-attachments/files/17778250/dani.s.vhs.preset.v.1.9.zip)
> 
> 
> - **v.2.0**  ▶ [dani's vhs preset v2.0.json](https://github.com/user-attachments/files/18239317/dani.s.vhs.preset.v2.0.json)
> 
> 
> - **v.2.1**  ▶ [dani's vhs preset v2.1.json](https://github.com/user-attachments/files/18495663/dani.s.vhs.preset.v2.1.json)
>
>
> - **v.2.2**  ▶ [dani's vhs preset v2.2.json](https://github.com/user-attachments/files/19110897/dani.s.vhs.preset.v2.2.json)
>
>
> - **v.2.3** ▶ [dani's vhs v.2.3.zip](https://github.com/user-attachments/files/25152607/dani.s.vhs.v.2.3.zip)
>
>
> - **v.2.4** [NEWEST] ▶ [dani's vhs preset v2.4.json](https://github.com/user-attachments/files/29681200/dani.s.vhs.preset.v2.4.json)

  ![image](https://github.com/user-attachments/assets/eb98091d-9676-4fa2-abcc-507e3e5b7007)

**Comment by Degamisu** [comment]
Oh wow! This is by far the best I've seen! This is an amazing representation on how the Handycam looks. Wonderful job, @notdigidani!

PS. What camera was used to take that video? Thanks!

**Reply by notdigidani** [reply]
thank you so much, @Degamisu!

and to answer your question, ive used my KODAK PIXPRO FZ101 to record the video. no problem!

**Reply by Degamisu** [reply]
ah, a beautiful camera. excellent work again, if possible can you send the original video?

**Reply by notdigidani** [reply]
sure thing

https://drive.google.com/file/d/1OTwNkhCtz3IVoCScGG_GVl7ZPv5LClGD/view?usp=sharing

**Reply by Degamisu** [reply]
> sure thing
> 
> https://drive.google.com/file/d/1OTwNkhCtz3IVoCScGG_GVl7ZPv5LClGD/view?usp=sharing

Thanks so much! Your video works very well on my black and white preset. heres a picture

![image](https://github.com/user-attachments/assets/ab8b6570-c1d4-4ad5-80e9-1ef82ac3f4ef)

**Reply by jazeelkhalid** [reply]
Hi @notdigidani, can we download this preset as a transparent .MOV file, which can be used a an overlay video file without any plugin.

**Comment by smfthe1199** [comment]
Wow, that's far more interesting and realistic. Thanks in advance.

**Reply by smfthe1199** [reply]
@notdigidani should I add this right off the bat in V0.1.3 of the NTSC-RT Ultimate Preset Pack? I don't feel that it's safe to do since people need to use the latest version till then.

**Reply by notdigidani** [reply]
you could add it but with a warning that said that you need the latest version or something. you have full permission to add my presets to the NTSC-RT Ultimate Preset Pack and do whatever you want with them, so do whatever suits you best

**Reply by smfthe1199** [reply]
YKW? I'll add it to the next version of it.

**Comment by oisintheblue** [comment]
yooo this new version (v1.9) is a huge glowup !!
keep up the good work

**Comment by kelfzz** [comment]
For the new version, is it just for some editing software (like davinci) because when i export it again, the video breaks (non davinci)

**Reply by notdigidani** [reply]
thats weird, i made on the standalone app

**Reply by kelfzz** [reply]
it freezes when i play it and then it closes

**Reply by kelfzz** [reply]
imma try playing it on vlc

**Reply by kelfzz** [reply]
it plays for a sec then freezes and skips the rest of the video

**Reply by kelfzz** [reply]
i got it to work i saved it seperately from the main

**Comment by mj4459675** [comment]
LEGNDRY!!!

**Reply by mj4459675** [reply]
the new handycam new 2.4

**Comment by jazeelkhalid** [comment]
What is this?
